CHANGELOG.md

```diff
@@ -36,5 +39,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - **Initial commit** - Forge source code, unittests, docs, pyproject.toml, README.md, and LICENSE files.
 
 [unreleased]: https://github.com/carsdotcom/cars-forge/compare/v1.0.1...HEAD
```

Comment:
This should be updated every time as well. We want to compare the unreleased changes to whatever the latest version is:
```suggestion
[unreleased]: https://github.com/carsdotcom/cars-forge/compare/v1.0.2...HEAD
```